python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -164,6 +170,8 @@ def __init__(
                 "is forbidden."
             )
 
+        self.filter_expr = filter_expr
```

Comment:
Would it be more readable to put it right after `super()` before we apply function like `_validate()`? Also note that functions such as `_validate()` in general might need to check arbitrary properties of this instance.